src/main/java/com/zombiecastlerush/building/Inventory.java

```diff
@@ -1,7 +1,27 @@
 package com.zombiecastlerush.building;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * TODO: a Shop, a Room or a Player can HAS-A Inventory
  */
-class Inventory {
+public class Inventory {
+    private List<Item> items = new ArrayList<>();
+
+    public Inventory(){
+    }
+
+    public List<Item> getItems(){
+        return items;
+    }
+
+    public void addItems(Item item){
```

Comment:
Oh, are we going to do a varargs here or an array of Items as the parameters for this method? Or is it just going to be addItem one by one? @wang-xiaotian, @gtulkn12, @kaashis

Reply:
varargs sounds good. I will update it.

Reply:
This is a default/empty constructor.

Reply:
added varargs